Bindings/Python/setup.py

```diff
@@ -15,7 +15,9 @@
       url='http://opensim.stanford.edu/',
       license='Apache 2.0',
       packages=['opensim'],
-      package_data={'opensim': ['_*.*']},
+      # The last 3 entries are for if OPENSIM_PYTHON_STANDALONE is ON.
+      # The asterisk after the extension is to handle version numbers on Linux.
```

Comment:
*Asterisk* is the correct spelling, though I usually pronounce the word more like "astrick".